scipy/misc/__init__.py

```diff
@@ -55,7 +60,6 @@
 __all__ = ['who', 'source', 'info', 'doccer', 'pade',
```

Comment:
If we're not importing `doccer`, we should also remove it from `__all__`.

Reply:
Good catch, I got tricked by my ``pyflakes`` checker. Restored.